Makefile

```diff
@@ -20,14 +20,44 @@ MAN_PAGES = $(shell ls $(MAN_DIR)/*.8)
 MAN_PAGES_BASE = $(notdir $(MAN_PAGES))
 MAN_INSTALL_PATH := ${PREFIX}/share/man/man8/
 
+RELEASE_DIR := $(CURDIR)/release
+
 VERSION := ${shell cat ./VERSION}
 
+SHELL := /bin/bash
```

Comment:
@cyphar doesn't use bash, as well as some other users, so I'm not sure if this is the right approach, I would rather we figure out why the command needs bash, maybe we should write some POSIX-shell-compatible commands.

Anyway if it runs well on machines without bash, I'm ok we merge it for now, @cyphar can you confirm it?

BTW, I would use `SHELL  ?= $(shell command -v bash 2>/dev/null)` if we want to define shell.